app/models/ability.rb

```diff
@@ -187,6 +188,7 @@ def signed_in_with_cfp_role(user)
     # ids of all the conferences for which the user has the 'cfp' role
     conf_ids_for_cfp = Conference.with_role(:cfp, user).pluck(:id)
 
+    can [:index, :show, :update], Resource, conference_id: conf_ids_for_cfp
```

Comment:
Well it is more readable. And `update` is also an alias, as it really means `update` + `edit`. And all this is done on purpose in the gem.

Anyway, it is not that important, I'm fine with this as it is. It is already like that in other parts of the ability file.